packages/nextcloud/lib/src/nextcloud.openapi.dart

```diff
@@ -10,68 +10,11 @@ import 'package:built_value/serializer.dart';
 import 'package:built_value/standard_json_plugin.dart';
 import 'package:dynamite_runtime/content_string.dart';
 import 'package:dynamite_runtime/http_client.dart';
-import 'package:universal_io/io.dart';
 
 export 'package:dynamite_runtime/http_client.dart';
 
 part 'nextcloud.openapi.g.dart';
 
-class NextcloudResponse<T, U> extends DynamiteResponse<T, U> {
-  NextcloudResponse(
-    super.data,
-    super.headers,
-  );
-
-  @override
-  String toString() => 'NextcloudResponse(data: $data, headers: $headers)';
```

Comment:
same as below.
The toString cuttently contains the prefix wich could make debugging easier.